main.go

```diff
@@ -71,14 +72,33 @@ func main() {
 		log.Event(ctx, "could not create the s3 client", log.ERROR, log.Error(err))
 	}
 
+	// Create Mongo client.
```

Comment:
Most other apps have a `service` package in order to remove as much as possible from `main.go`. This app doesn't as it's quite an old app now (comparatively) that predates the service package approach. I wonder if now is the time to do some refactoring before we put too much in `main`.